openapi/endpoints/login/models/challenge-request.yaml

```diff
@@ -2,18 +2,36 @@ type: object
 required:
   - walletAddress
   - chainId
+  - clientApplicationId
+  - scopes
 properties:
   walletAddress:
     description: The address of the signing wallet or the smart contract that implements ERC-1271
     type: string
-    example: "0xA3058369d6A481B1ff08F62B352409c3D709De9b"
+    example: '0xA3058369d6A481B1ff08F62B352409c3D709De9b'
   chainId:
     description: Chain ID (see https://chainlist.org/)
     type: number
     example: 1
+  clientApplicationId:
+    description: Client application ID assigned by IMMERSVE
```

Comment:
"ID of the client application created in an Immersve partner profile"

Reply:
but not sure, up to you

Reply:
updated